docs/.vuepress/theme/styles/index.styl

```diff
@@ -177,7 +177,6 @@ a {
 table {
   width: 100%;
   display: table;
-  word-break: break-word;
```

Comment:
It is required for responsiveness: see  #8296, #8341 

Should we attempt to turn it off only on mobile? Have any idea?

Reply:
From my observation, only very wide tables look ugly. That's why I decided to style the table differently only for guides pages.

Reply:
how about tables listed by @aninde ? 
https://github.com/handsontable/handsontable/issues/8296

Reply:
Thanks! I've checked that and improved the style by adding media queries for narrow viewports c0fe37bbd1e76f9ae40ec1eb7a5818ffd41abeff. I think that now, all or at least most of the cases are covered.